test_0 (__main__.lab1a.test_0)
[Lab 1] - [Investigation 3] - [Part 2] - printing - Test for file creation: ./lab1a.py ... ok
test_a (__main__.lab1a.test_a)
[Lab 1] - [Investigation 3] - [Part 2] - printing - Test for errors running: ./lab1a.py ... /usr/lib64/python3.12/subprocess.py:822: ResourceWarning: unclosed file <_io.TextIOWrapper name=3 encoding='UTF-8'>
  _cleanup()
ResourceWarning: Enable tracemalloc to get the object allocation traceback
ok
test_a1 (__main__.lab1a.test_a1)
[Lab 1] - [Investigation 3] - [Part 2] - printing - Test for correct shebang line: ./lab1a.py ... ok
test_b (__main__.lab1a.test_b)
[Lab 1] - [Investigation 3] - [Part 2] - printing - Test output for correct output "Hello world": ./lab1a.py ... ok
test_0 (__main__.lab1b.test_0)
[Lab 1] - [Investigation 4] - string objects & printing - Test for file creation: ./lab1b.py ... ok
test_a (__main__.lab1b.test_a)
[Lab 1] - [Investigation 4] - string objects & printing - Test for errors running: ./lab1b.py ... ok
test_a1 (__main__.lab1b.test_a1)
[Lab 1] - [Investigation 4] - string objects & printing - Test for correct shebang line: ./lab1b.py ... ok
test_b (__main__.lab1b.test_b)
[Lab 1] - [Investigation 4] - string objects & printing - Test for correct output "How old are you Isaac?": ./lab1b.py ... FAIL

======================================================================
FAIL: test_b (__main__.lab1b.test_b)
[Lab 1] - [Investigation 4] - string objects & printing - Test for correct output "How old are you Isaac?": ./lab1b.py
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/kushagra/ops445/lab1/./CheckLab1.py", line 126, in test_b
    self.assertEqual(stdout, expected_output, msg=error_output)
AssertionError: b'Isaac is 72 years old!\n' != b'How old are you Isaac?\n' : output is not correct(HINT: pay attention to uppercase letters, spaces, and punctuation)

----------------------------------------------------------------------
Ran 8 tests in 0.807s

FAILED (failures=1)
OPS445 Lab Report - System Information for running ./CheckLab1.py
=================================================================
    User login name: kushagra
    Git Email: ksingh631@myseneca.ca
    Linux system name: fedora
    Python executable: /usr/bin/python3
    Python version:  3.12.6 (main, Sep  9 2024, 00:00:00) [GCC 14.2.1 20240801 (Red Hat 14.2.1-1)]
    OS Platform: linux
    Working Directory: /home/kushagra/ops445/lab1
    Start at: Fri Sep 20 14:21:57 2024
=================================================================
